lib/ui/views/login_view.dart

```diff
@@ -123,10 +142,17 @@ class _LoginViewState extends State<LoginView> {
                                       (Set<MaterialState> states) {
                                         if (states
                                             .contains(MaterialState.disabled)) {
-                                          return Colors.white38;
+                                          return getButtonColor(
+                                              model.getAppTheme(),
+                                              context,
+                                              Colors.white38,
+                                              Colors.grey[800]);
                                         }
-                                        return Colors
-                                            .white; // Use the component's default.
+                                        return getButtonColor(
+                                            model.getAppTheme(),
+                                            context,
+                                            Colors.white,
+                                            Colors.grey[900]);
```

Comment:
```suggestion
                                        return buttonColor;
```

If you apply the fixes below 👇  😄